app/templates/user/email-verified-view.jade

```diff
@@ -0,0 +1,36 @@
+extends /templates/base-flat
+
+block content
+  .container.text-center
+    if state.get('verifyStatus') === "pending"
+      span(data-i18n="account.verifying_email")
+        | Verifying your email address...
+    else if state.get('verifyStatus') === "success"
+      .alert.alert-success.center-block
+        .fa.fa-check-circle-o
+        // TODO: Install font-awesome
```

Comment:
We should use glyphicons for now, and switch if and when we decide to install font-awesome.